test/smoke.spec.js

```diff
@@ -154,4 +160,49 @@ describe("Smoke tests", () => {
       });
     });
   });
+
+  describe("daemon", () => {
+    beforeEach(() => mkdirp(testOutputDir));
```

Comment:
NIT: All file i/o, even in tests should be async unless "overriding reasons". Here, it's trivial to do async, so let's do that for mkdir and rimraf

Reply:
`mkdirp` returns a Promise, so it should already be async–but `rimraf` uses callbacks and I got lazy 😄 I'll throw a `done` in there for that one